flink-core/src/main/java/org/apache/flink/api/common/io/FinalizeOnMaster.java

```diff
@@ -66,6 +66,7 @@ interface FinalizationContext {
          *
          * @param subtaskIndex the subtask index.
          * @return the finished attempts.
+         * @throws IllegalArgumentException Thrown, if subtaskIndex is invalid.
```

Comment:
This change should be part of a previous commit. This also applies some other change in this commit.

Reply:
Oops, you are right. I squash the wrong commit.